api/python/setup.py

```diff
@@ -40,6 +40,11 @@
     'pytest-xdist==1.29.0',
     'pytest-timeout==1.3.4',
     'pytest-mock==3.1.0',
+    'attrs==19.1.0',
```

Comment:
no, please remove that, attrs is [part of our repo](https://github.com/Seagate/cortx-prvsnr/tree/pre-cortx-1.0/api/python/provisioner/vendor/attr) actually

Reply:
Done